### PR TITLE
FIX: Revise *Psychopy*'s installation

### PR DESCRIPTION
**Pull request by oesteban**
I thought I had already submitted this. This basically resolves the issues with *Psychopy*'s installation last week. It also writes down how to load the environment (apologies, @acionca, because I thought this was already in yesterday).

Integrates https://github.com/TheAxonLab/hcph-sops/issues/253#issuecomment-1765928854

### CODE REVIEW

File: docs/data-collection/pre-session.md [hunk 1]
```diff
@@ -22,17 +22,29 @@ Instructions of operations to be performed before the participant arrival, **bef
...
+        ```
+
+- [ ] Open a terminal and move it to the secondary screen
+- [ ] Run the compiled tasks with *Python* directly.
 - [ ] Load in the different experiments and check for proper functioning:
-    - [ ] `{{ settings.psychopy.tasks.func_rest }}` (resting-state fMRI):
+    - [ ] `python {{ settings.psychopy.tasks.func_rest }}.py` (resting-state fMRI):
         - [ ] check that the movie is played.
-    - [ ] `{{ settings.psychopy.tasks.func_bht }}` (breath-holding task, BHT):
+    - [ ] `{{ settings.psychopy.tasks.func_bht }}.py` (breath-holding task, BHT):
```
Comment (esavary) [inline docs/data-collection/pre-session.md]:
```suggestion
    - [ ] `python {{ settings.psychopy.tasks.func_bht }}.py` (breath-holding task, BHT):
```

File: docs/data-collection/pre-session.md [hunk 2]
```diff
@@ -22,17 +22,29 @@ Instructions of operations to be performed before the participant arrival, **bef
...
         - [ ] check that the task runs properly.
-    - [ ] `{{ settings.psychopy.tasks.func_qct }}` (quality-control task, QCT):
+    - [ ] `{{ settings.psychopy.tasks.func_qct }}.py` (quality-control task, QCT):
```
Comment (esavary) [inline docs/data-collection/pre-session.md]:
```suggestion
    - [ ] `python {{ settings.psychopy.tasks.func_qct }}.py` (quality-control task, QCT):
```

File: docs/data-collection/scanning.md [hunk 3]
```diff
@@ -257,7 +267,12 @@ It is critical to check that physiological signals are looking good:
 - [ ] While the fieldmap sequence is running,
     - [ ] [Adjust the FoV](scanning-notes.md#setting-the-fov) for the positive-control-task (`func-bold_task-qct_dir-{RL,LR,PA,AP}__cmrr_me4_sms4`) fMRI sequence, and
     - [ ] verify the *Number of measurements* with respect to the [task's timing](intro.md#task-timing) ({{ settings.mri.timings.func_qct }}).
-    - [ ] Verify that the positive-control task `{{ settings.psychopy.tasks.func_qct }}` is open in psychopy, that you calibrated the ET and that the physiological signals are still recording and look ok.
+    - [ ] with a terminal open on the projector's screen, execute the QCT:
```
Comment (esavary) [inline docs/data-collection/scanning.md]: Yesterday's test confirmed that we don't have to open the terminal on the projector screen. We just have to keep the mouse cursor within the projector screen when launching the task.

**Review by esavary — CHANGES_REQUESTED** [review]

File: docs/data-collection/pre-session.md (hunk 1 above)
Comment (oesteban) [reply docs/data-collection/pre-session.md]: ~~This refers to the file itself... I'm inclined not to put python here without making sure the environment is correct. Please fix in a future PR :)~~

Changed my mind, see below.

File: docs/data-collection/scanning.md (hunk 3 above)
Comment (oesteban) [reply docs/data-collection/scanning.md]: okay, I'm starting a major overhaul of this PR, you are right

File: docs/data-collection/pre-session.md (hunk 1 above)
Comment (oesteban) [reply docs/data-collection/pre-session.md]: Okay, I fixed here.

**Comment by oesteban** [comment]
@acionca @esavary care for a speedy review?

**Comment by acionca** [comment]
> @acionca @esavary care for a speedy review?

I can have a look and test things during the session if it's ok

**Comment by oesteban** [comment]
> > @acionca @esavary care for a speedy review?
> 
> I can have a look and test things during the session if it's ok

Let's do one thing - we merge it now, and during the session, you open issues with the problems you encounter.

**Comment by acionca** [comment]
> Let's do one thing - we merge it now, and during the session, you open issues with the problems you encounter. 

Sounds good !

**Pull request merged**